connector/src/main/java/org/jboss/as/connector/util/RaServicesFactory.java

```diff
@@ -89,6 +91,17 @@ public static void createDeploymentService(final ManagementResourceRegistration
                 .addDependency(NamingService.SERVICE_NAME)
                 .addDependency(ConnectorServices.BOOTSTRAP_CONTEXT_SERVICE.append(bootStrapCtxName))
                 .addDependency(ConnectorServices.RESOURCE_ADAPTER_DEPLOYER_SERVICE_PREFIX.append(connectorXmlDescriptor.getDeploymentName()));
+        String raName = deployment;
+        if (raxml.getId() != null) {
+            raName = raxml.getId();
+        }
+        ServiceName parentName = ServiceName.of(ConnectorServices.RA_SERVICE, raName);
+        for(ServiceName subServiceName: serviceRegistry.getServiceNames()) {
+            if (parentName.isParentOf(subServiceName)
```

Comment:
Yes, the service with name: `parentName` holds the value of `Activation`, which  is used by the deployer service to do the deployment, so it needs to depend on this service.

In case of any child service like: `ConfigPropertiesService` available, this service(`parentName`) is an indirect dependency already, but I think it should be fine.